lib/elastic_apm/config/options.rb

```diff
@@ -48,6 +48,7 @@ def normalize(val)
           when :bool then normalize_bool(val)
           when :list then normalize_list(val)
           when :dict then normalize_dict(val)
+          when :url then normalize_trailing_slash(val)
```

Comment:
Maybe align with the other types and call it normalize_url

Reply:
good point, fixed!